### ISSUE TITLE
Pressing NVDA+f appears to deselect text in Word 2016

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#5799]
In Word 2016, pressing NVDA+f or NVDA+up arrow when text is selected do what they should, however the selected text appears to be deselected.  Text is still selected - if you delete or underline etc, it affects the selected text.  If you extend the selection, the new and originally selected text once more all appear selected.

NVDA+shift+up arrow also confirms selection even when not visible.

Seems to happen 100% of the time with NVDA+f.
With NVDA+up arrow, if you press it once, the selection disappears.  If you press it twice quickly (so it spells the selection), the highlight remains visible / returns.

I tried with a longer selection and it still needs to be quick enough to spell - doing it after say a second while it's still reading a longer selection doesn't make the selection reappear.

Could not reproduce in Excel or WordPad.

**Comment by feerrenrut** [comment]
Tried to reproduce this in Word 2013, I notice the selection flash but it continues to be selected after using NVDA+f and NVDA+up arrow.
 
 Using build: 13358,a989f5ee with Office 2013 Version 15.0.4823.1004

**Comment by ehollig** [comment]
Maybe I am not doing it right, but when selecting a paragraph of text in Microsoft Word 2016 and performing NVDA+F, text remains to appear selected visually.

**Comment by feerrenrut** [comment]
@Qchristensen are you still able to reproduce this?

**Comment by Qchristensen** [comment]
Using either 2017.2, or Next 14189 or Master 14175, I can still reproduce this on Office 2016 on Windows 10, yes.  If @ehollig can't reproduce it, could it be something local, such as a specific video driver / hardware?  I have an NVIDIA GeForce GTX 745 on a Samsung 32" monitor (although just looking at Device manager, I'm just using the "Generic PnP monitor" driver.

**Comment by feerrenrut** [comment]
Given that Word still behaves as if it has the text selected, but visually it appears not to, it seems like this might be a rendering bug in Word. This is probably being triggered by the way that we update selections in MS Word to gather various bits of information. Ideally we should try to find the set of calls that causes this and report a bug to Microsoft. However, there is a plan to switch to UIA in the future which would remove the need for all this selecting / unselecting code (see #7409).

Regardless this is a serious issue, giving this a priority 2.

**Comment by Adriani90** [comment]
@Qchristensen are you still able to reproduce this issue?

**Comment by Qchristensen** [comment]
Yes, I can still reproduce this.  I should clarify my earlier comment: I am NOT running Office 2016, but Office 365.

**Comment by Qchristensen** [comment]
I just realised, I was running NVDA 2018.4.1 when I tested last.

Using NVDA alpha-16793,31692c6c, In Office 365 Version: 16.0.11231.20164, I can no longer reproduce the original issue.

My experience currently is the same as Reef's - that is, the selection flashes for a moment, but stays (visually) selected.

**Comment by Adriani90** [comment]
Closing as works for me in last Alpha. If the issue still occurs though, please comment on it and we can reopen it. Thanks.
